### ISSUE TITLE
[ADD] Option to delete old wallets 

### ISSUE THREAD

**Issue by HolyK1cks** [casinocoin/tokenwallet-mobile#146]
Would like a way to delete old wallets if they aren't in use anymore. Ex: 
![Screenshot_20200206-094834_Twitter](https://user-images.githubusercontent.com/57190578/73953469-0e5e3180-48c6-11ea-8f1a-a8beb63effa6.jpg)

Say I want to delete the bottom 2 wallets since I only want to use the first one. Possibly have a way to hide them if you can't delete them?

**Comment by t3nshi** [comment]
well, actually we had this discussion many times, and there is a thing to remark here is that the app should hold information consistent with the ledgers on the blockchain, and there is no way to delete -and there shouldn't be-  any account already on the blockchain. 

I think it is worth considering the alternative you mention, maybe a setting that could be enabled to hide accounts on the wallet. They will be there, never completely gone, but they won't create unnecessary clutter when I don't want them getting in the way.

**Comment by HolyK1cks** [comment]
Yeah, dont necessarily need to delete them, but a "hide" option would be nice!
